src/models.js

```diff
@@ -152,7 +154,14 @@ async function sessionRun(session, inputs) {
     } catch (e) {
         // This usually occurs when the inputs are of the wrong type.
         console.error(`An error occurred during model execution: "${e}".`);
-        console.error('Inputs given to model:', checkedInputs);
+        // Not log full data, it may cause crash in React Native
+        console.error(
+            'Inputs given to model:',
+            Object.fromEntries(
+                Object.entries(checkedInputs)
+                    .map(([key, tensor]) => [key, { dims: tensor.dims, type: tensor.type }])
+            )
+        );
```

Comment:
Good idea. When we add debug/logging levels, we can log all output in "verbose" mode, and this normally perhaps. Or maybe default to full output in browser env (since the browser outputs it nicely)

Reply:
This change just because full tensor log on RN will cause error (even crash)

Reply:
I add check to log full tensor on browser